src/parcsr_mv/par_csr_matop.c

```diff
@@ -6695,39 +6695,39 @@ hypre_ParCSRMatrixBlockColSumHost( hypre_ParCSRMatrix     *A,
    /* Communication variables */
    hypre_ParCSRCommPkg    *comm_pkg          = hypre_ParCSRMatrixCommPkg(A);
    HYPRE_Int               num_sends         = hypre_ParCSRCommPkgNumSends(comm_pkg);
-   HYPRE_Int              *send_map_elmts    = hypre_ParCSRCommPkgSendMapElmts(comm_pkg);
-   HYPRE_Int              *send_map_starts   = hypre_ParCSRCommPkgSendMapStarts(comm_pkg);
+   HYPRE_Int              *send_map_elmts;
+   HYPRE_Int              *send_map_starts;
 #if defined(HYPRE_USING_PERSISTENT_COMM)
    hypre_ParCSRPersistentCommHandle *comm_handle;
 #else
    hypre_ParCSRCommHandle           *comm_handle;
 #endif
 
    /* Update commpkg offsets */
-   hypre_ParCSRCommPkgUpdateVecStarts(comm_pkg, num_cols_block_B, 1, num_cols_block_B);
```

Comment:
Why is this no longer needed?

Reply:
I simplified the previous implementation of this function and this line became:
`hypre_ParCSRCommPkgUpdateVecStarts(comm_pkg, 1, 0, 1);`